tellopy/_internal/tello.py

```diff
@@ -420,15 +423,31 @@ def set_roll(self, roll):
             log.info('set_roll(val=%4.2f)' % roll)
         self.right_x = self.__fix_range(roll)
 
+    def set_fast_mode(self):
```

Comment:
Why does set_fast_mode API take no argument and it toggles the mode internally?
I don't know what is the "fast mode" and the effect of the mode.
Is the toggle action appropriate for the mode?
I feel that you should give an argument which specify the mode in set_fast_mode() API.

Reply:
Agreed 'set_fast_mode' should match it's action. Hence, I changed the function to 'toggle_fast_mode'. Furthermore, no argument should be given since enabling 'fast_mode' for the Tello is done by changing a single bit from 0 to 1 within the 'STICK_CMD' msg.

Effect by fast mode is faster responsiveness by the on-board controller and higher fly velocities.